etc/org.opencastproject.distribution.aws.s3.AwsS3DistributionServiceImpl.cfg

```diff
@@ -47,9 +47,20 @@ org.opencastproject.distribution.aws.s3.distribution.enable=false
 
 # Enable presigned URL
 # Leave this commented out to use presigned URL fow AWS S3
```

Comment:
I think this is not true. Presigned URLs are not used by default.

```suggestion
```

Reply:
True, I fixed the value, but missed the comment. Thx!